lib/plugins/crowdsec/utils.lua

```diff
@@ -9,13 +9,21 @@ M.HTTP_CODE["301"] = ngx.HTTP_MOVED_PERMANENTLY
 M.HTTP_CODE["302"] = ngx.HTTP_MOVED_TEMPORARILY
 M.HTTP_CODE["400"] = ngx.HTTP_BAD_REQUEST
 M.HTTP_CODE["401"] = ngx.HTTP_UNAUTHORIZED
-M.HTTP_CODE["401"] = ngx.HTTP_UNAUTHORIZED
 M.HTTP_CODE["403"] = ngx.HTTP_FORBIDDEN
 M.HTTP_CODE["404"] = ngx.HTTP_NOT_FOUND
 M.HTTP_CODE["405"] = ngx.HTTP_NOT_ALLOWED
 M.HTTP_CODE["406"] = ngx.HTTP_NOT_ACCEPTABLE
 M.HTTP_CODE["500"] = ngx.HTTP_INTERNAL_SERVER_ERROR
 
+M.CONTENT_TYPE = {}
```

Comment:
I'm very sorry for the very late review. Is there any reason not to allow users to set `content-type` directly? I'm afraid the mapping is more error-prone than anything.